webapp/components/interop.html

```diff
@@ -324,6 +328,10 @@ <h2>Interoperability</h2>
       stateFunc(state, '', `/interop${path}?q=${encodedQuery}`);
     }
 
+    computeTestScheme(path) {
+      return path.indexOf('.https.') !== -1 ? 'https' : 'http';
```

Comment:
A link to the logic in wpt run which this needs to be in sync with would be nice. I'd be surprised if there are any mismatches even if the rule isn't exactly the same, so not asking to centralize the definition or anything :)

Reply:
Added.